python/conda_env_requirements_ctsm_pylib.txt

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
The above text files that are deleted should be put back and the files below added should be removed.
And those changes go on the branch.